backend/test_observer/controllers/test_executions/models.py

```diff
@@ -81,6 +85,30 @@ class EndTestExecutionRequest(BaseModel):
 
 
 class TestExecutionsPatchRequest(BaseModel):
-    c3_link: HttpUrl | None = None
-    ci_link: HttpUrl | None = None
-    status: TestExecutionStatus | None = None
+    c3_link: HttpUrl | None
+    ci_link: HttpUrl | None
+    status: TestExecutionStatus | None
```

Comment:
any reason for removing optional parameters in the old patch request?

Reply:
I think git detected this as a change because I haven't merged main. Will make sure to check it after merging main.